packages/renderer/src/lib/configmaps-secrets/ConfigMapSecretColumnActions.svelte

```diff
@@ -2,7 +2,11 @@
 import ConfigmapSecretActions from './ConfigMapSecretActions.svelte';
 import type { ConfigMapSecretUI } from './ConfigMapSecretUI';
 
-export let object: ConfigMapSecretUI;
+let {
+  object,
+}: {
+  object: ConfigMapSecretUI;
+} = $props();
```

Comment:
AFAIK we use a Props interface when declaring properties now

Reply:
We do? We have both in the codebase and at least some of this style weren't from me. 😀 @axel7083 do you have a preference?

Reply:
we started without and switched to Props as it's suggested by the svelte documentation

https://svelte.dev/docs/svelte/typescript#The-Component-type

Reply:
Yes, this is my preference, but I am not blocking it (since it is not in the code guidelines yet)

My 2 cents would be to go further.

```
.
└── table/
    └── columns/
        ├── Foo.svelte
        ├── Bar.svelte
        └── props.ts
```

Instead of redefining the same props again and again, since all the columns for a given table have the same, we should group them in a folder, create a `props.ts` file with the props definition in it, that each component should import and use. :)

Reply:
Like this?

<img width="302" alt="Screenshot 2025-02-07 at 8 57 30 AM" src="https://github.com/user-attachments/assets/524b90d4-7297-4893-ada3-c5c9ac60ebc8" />

Yeah, I like it better. I'd rather commit this and do it separately though, so I opened #11035.

Might make sense to do something similar for the details pages in the future too.

Reply:
yeah, follow-up PR! As I said this is more like improvement area!